app/_src/guides/consumer-producer-policies.md

```diff
@@ -0,0 +1,262 @@
+Producer policies will be created in the same zone as MeshService they target. Producer policies will be synced to other zones. 
+
+**Consumer** is someone that utilizes a service by communicating with it. Consumer policies will be applied in consumer 
+namespace and will target MeshService from different namespace. Consumer policy will take effect only in consumer namespace.
+Consumer policies will override producer policies.
+
+## Testing producer policy
+
+To test MeshTimeout that we've applied in previous steps we need to simulate delays on our requests. To do this we need
+MeshFaultInjection policy
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshFaultInjection
+metadata:
+  name: default
+  namespace: kuma-demo
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+    proxyTypes: [Sidecar]
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        http:
+          - delay:
+              percentage: '50'
+              value: 2s" | kubectl apply -f -
+```
+
+We can now make few requests to our demo-app, and we should see timeouts:
+
+```shell
+kubectl exec -n first-consumer consumer -- curl --no-progress-meter demo-app.kuma-demo.mesh:5000/counter
+```
+
+Example output:
+
+```
+{"counter":"0","zone":"local","err":null}
+upstream request timeout
+```
+
+We should see the same results when making requests from second-consumer namespace:
+
+```shell
+kubectl exec -n second-consumer consumer -- curl --no-progress-meter demo-app.kuma-demo.mesh:5000/counter
+```
+
+## Utilizing consumer policy
+
+To this moment we were relying on producer policy created previously. Let's assume that we don't mind waiting a little longer
+for the response from demo-app, and we would like to override the timeout just for our namespace. To do so we need to create new policy:
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: consumer-timeout
+  namespace: first-consumer
+  labels:
+    kuma.io/mesh: default
+    kuma.io/origin: zone
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: demo-app
+        namespace: kuma-demo
+      default:
+        http:
+          requestTimeout: 3s" | kubectl apply -f -
+```
+
+When we now make requests from the first-consumer namespace all the requests should succeed, but they will take longer:
+
+```shell
+kubectl exec -n first-consumer consumer -- curl --no-progress-meter demo-app.kuma-demo.mesh:5000/counter
+```
+
+Policy that we have just applied is consumer policy. This means that traffic from other namespaces to demo-app should not be affected by it.
+We can test this by making requests from our second-consumer namespace:
+
+```shell
+kubectl exec -n second-consumer consumer -- curl --no-progress-meter demo-app.kuma-demo.mesh:5000/counter
+```
+
+We should still see timeouts:
+
+```
+upstream request timeout
+```
```

Comment:
@slonka in his latest guide has added a `Things to remember section` maybe a general `What we've learnt?` would be useful here?

Reply:
added `What we've learnt?` section